<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SM_MainWindow</class>
 <widget class="QMainWindow" name="SM_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Storage Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="StorageMap_View">
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foregroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="acFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="acFitToPage"/>
    <addaction name="acZoomIn"/>
    <addaction name="acZoomOut"/>
    <addaction name="acGrid"/>
    <addaction name="acBBox"/>
   </widget>
   <widget class="QMenu" name="menuGraph_Edit">
    <property name="title">
     <string>Graph Edit</string>
    </property>
    <addaction name="acSelectAll"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuGraph_Edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolMain">
   <property name="windowTitle">
    <string>Scene tools</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acFitToPage"/>
   <addaction name="acZoomIn"/>
   <addaction name="acZoomOut"/>
   <addaction name="acGrid"/>
   <addaction name="acBBox"/>
   <addaction name="acInfoRails"/>
   <addaction name="acMainRail"/>
  </widget>
  <widget class="QToolBar" name="toolEdit">
   <property name="windowTitle">
    <string>Graph tools</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dkObjectProps">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Object Props</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTableView" name="tvObjectProps">
       <property name="minimumSize">
        <size>
         <width>325</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView::item:focus
{
border : 2px solid blue;
/*color: white;*/
/*selection-color: black;
selection-background-color: red;*/
}
</string>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkView">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>Scene preferences</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
         <item>
          <widget class="QLabel" name="lbGridSize">
           <property name="text">
            <string>Grid size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbGridSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>1000000000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolBar" name="toolMain">
         <property name="windowTitle">
          <string>Tool Bar</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <addaction name="acGrid"/>
         <addaction name="acBBox"/>
         <addaction name="acSpecialLines"/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="acFitToPage">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Common/images/zoom_fit.png</normaloff>:/Common/images/zoom_fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit To Page</string>
   </property>
   <property name="toolTip">
    <string>Fit To Page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="acZoomIn">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Common/images/zoom_in.png</normaloff>:/Common/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="acZoomOut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Common/images/zoom_out.png</normaloff>:/Common/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="acGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Common/images/grid.png</normaloff>:/Common/images/grid.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Grid</string>
   </property>
   <property name="shortcut">
    <string>#</string>
   </property>
  </action>
  <action name="acBBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Common/images/bbox.png</normaloff>:/Common/images/bbox.png</iconset>
   </property>
   <property name="text">
    <string>BBox</string>
   </property>
   <property name="toolTip">
    <string>BBox</string>
   </property>
  </action>
  <action name="acInfoRails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Common/images/info_rails.png</normaloff>:/Common/images/info_rails.png</iconset>
   </property>
   <property name="text">
    <string>Show/hide info rails</string>
   </property>
   <property name="toolTip">
    <string>Show/hide info rails</string>
   </property>
  </action>
  <action name="acMainRail">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Common/images/main_rail.png</normaloff>:/Common/images/main_rail.png</iconset>
   </property>
   <property name="text">
    <string>Show/hide main rail</string>
   </property>
   <property name="toolTip">
    <string>Show/hide main rail</string>
   </property>
  </action>
  <action name="acSelectAll">
   <property name="text">
    <string>Select all items</string>
   </property>
   <property name="toolTip">
    <string>Select all items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="acSpecialLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Common/images/special_lines.png</normaloff>:/Common/images/special_lines.png</iconset>
   </property>
   <property name="text">
    <string>Storages rotate lines</string>
   </property>
   <property name="toolTip">
    <string>Show storages rotate lines</string>
   </property>
  </action>
  <action name="acFullScreen">
   <property name="text">
    <string>Full screen</string>
   </property>
   <property name="toolTip">
    <string>Full screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
